webpack.config.js

```diff
@@ -198,10 +198,7 @@ module.exports = function (env = {}) {
       hot: false,
       liveReload: false,
       client: {
-        overlay: {
-          warnings: false,
-          errors: true,
-        },
+        overlay: false,
```

Comment:
todo: probably remove, find fix or merge intentionally.

Reply:
for the record, we agreed to remove the overlay, so this todo is resolved :)